MatrixSDK/Background/Crypto/MXBackgroundCryptoV2.swift

```diff
@@ -60,6 +60,8 @@ class MXBackgroundCryptoV2: MXBackgroundCrypto {
           - to-device events : \(syncResponse.toDevice?.events.count ?? 0)
           - devices changed  : \(syncResponse.deviceLists?.changed?.count ?? 0)
           - devices left     : \(syncResponse.deviceLists?.left?.count ?? 0)
+          - one time keys    : \(syncResponse.deviceOneTimeKeysCount?[kMXKeySignedCurve25519Type] ?? 0)
+          - fallback keys    : \(syncResponse.unusedFallbackKeys ?? [])
```

Comment:
I'm sure its not, but double checking: Is this going to log keys that are sensitive?

Reply:
Those keys are public, after all they get uploaded to the server.

Reply:
Yea, actually in this particular log we only print the count of keys (or type in the case of fallback keys), but I have also started logging the actual keys elsewhere so your question is valid (and it is safe as we only print hte public keys)